sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample10_PublishAnEventBatchWithCustomSizeLimit.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Samples.Infrastructure;
+
+namespace Azure.Messaging.EventHubs.Samples
+{
+    /// <summary>
+    ///   An example of publishing events using a custom size limitation with the batch.
+    /// </summary>
+    ///
+    public class Sample10_PublishAnEventBatchWithCustomSizeLimit : IEventHubsSample
+    {
+        /// <summary>
+        ///   The name of the sample.
+        /// </summary>
+        ///
+        public string Name { get; } = nameof(Sample10_PublishAnEventBatchWithCustomSizeLimit);
+
+        /// <summary>
+        ///   A short description of the sample.
+        /// </summary>
+        ///
+        public string Description { get; } = "An example of publishing events using a custom size limitation with the batch.";
+
+        /// <summary>
+        ///   Runs the sample using the specified Event Hubs connection information.
+        /// </summary>
+        ///
+        /// <param name="connectionString">The connection string for the Event Hubs namespace that the sample should target.</param>
+        /// <param name="eventHubName">The name of the Event Hub, sometimes known as its path, that she sample should run against.</param>
+        ///
+        public async Task RunAsync(string connectionString,
+                                   string eventHubName)
+        {
+            // We will start by creating a producer client using its default options.
+
+            await using (var producerClient = new EventHubProducerClient(connectionString, eventHubName))
+            {
+                // There is a limit to the size of an event or batch of events that can be published at once.  This limit varies and depends
+                // on the Event Hubs service and the properties that the namespace of the target Event Hub.  Because the size limit is based
```

Comment:
`depends on the Event Hubs service and the properties that the namespace of the target Event Hub`. Did you mean something like `and the properties of the namespace`?

Reply:
Good catch.   That was one heck of an awkward half-phrase.